CondTools/Geometry/test/calogeometrywriter.py

```diff
@@ -2,7 +2,8 @@
 
 process = cms.Process("CaloGeometryWriter")
 process.load('CondCore.DBCommon.CondDBCommon_cfi')
-process.load('Configuration.StandardSequences.GeometryExtended_cff')
+process.load('Configuration.Geometry.GeometryExtended_cff')
+process.load('Configuration.Geometry.GeometryReco_cff')
```

Comment:
@bsunanda - please, remove this line. All relevant producers are include through the next one with appropriate parameters. Besides, if this file is not modified, the PR will be merged faster.

Another question: does it mean Hcal geometry records in DB are wrong and need to be recreated? Thanks